pkg/prove/proof_test.go

```diff
@@ -188,13 +185,14 @@ func TestTxSharePosition(t *testing.T) {
 // 	assert.Equal(t, rawShares, genShares)
 // }
 
-func joinByteSlices(s ...[]byte) string {
-	out := make([]string, len(s))
-	for i, sl := range s {
-		sl, _, _ := shares.ParseDelimiter(sl)
-		out[i] = string(sl[appconsts.NamespaceSize+appconsts.ShareInfoBytes:])
+// stripCompactShares strips the universal prefix (namespace, info byte) and
+// reserved byte from a list of compact shares and joins them into a single byte
+// slice.
+func stripCompactShares(compactShares [][]byte, start uint64, end uint64) (result []byte) {
+	for i := start; i <= end; i++ {
+		result = append(result, compactShares[i][appconsts.NamespaceSize+appconsts.ShareInfoBytes+appconsts.CompactShareReservedBytes:]...)
```

Comment:
[for reviewers] note we don't need to parse the unit length delimiter that is prefixed to a transaction because the test checks that this share range contains the raw tx bytes. Therefore, it is safe to keep the unit length delimiter in the stripped compact shares returned from this function